packages/react-components/src/components/styles/TogetherMode.styles.ts

```diff
@@ -198,16 +198,26 @@ export const togetherModeParticipantDisplayName = (
   isParticipantHovered: boolean,
   participantSeatingWidth: number,
   color: string
-): CSSProperties => {
-  const MIN_DISPLAY_NAME_WIDTH = 100;
+): React.CSSProperties => {
+  const width =
+    isParticipantHovered || participantSeatingWidth * REM_TO_PX_MULTIPLIER > 100
+      ? 'fit-content'
+      : _pxToRem(0.7 * participantSeatingWidth * REM_TO_PX_MULTIPLIER);
+
+  const display =
+    isParticipantHovered || participantSeatingWidth * REM_TO_PX_MULTIPLIER > 150 ? 'inline-block' : 'none';
```

Comment:
Can we understand what this multiplication is comparing against? I imagine its the minimum sizing for a tile? lets give it a constant if we can